src/software/test_util/test_util.cpp

```diff
@@ -131,8 +132,8 @@ namespace TestUtil
         return states;
     }
 
-    GameState createGameState(const RefereeCommand &current_referee_command,
-                              const RefereeCommand &previous_referee_command)
+    GameState createGameStateProto(const RefereeCommand &current_referee_command,
```

Comment:
GameState is a c++ type, not a Proto type (a proto is any file found in software/proto), this should remain as the same

[GameState](https://github.com/UBC-Thunderbots/Software/blob/master/src/software/world/game_state.h)